tools/travis/run-tests.sh

```diff
@@ -12,7 +12,7 @@
 set -e
 
 # Run the data build suite (integration tests).
-$(npm bin)/cypress run --headless --browser chrome --config integrationFolder=cypress/tests/data
+$(npm bin)/cypress run --headless --browser chrome
```

Comment:
These changes will cause the database to be dumped *after* the integration tests are run, rather than between the data build and integration test stages. Please keep these running separately; you can use...
```
--config '{"specPattern":["cypress/tests/data/**/*.cy.js"]}'
```
...in place of...
```
--config integrationFolder=cypress/tests/data
```
...and something similar below for `tests/integration`.

Reply:
I've resolved this (and converted the pkp-lib tests and a few other things) and merged in separate PRs.